RELEASE_NOTES.md

```diff
@@ -1,5 +1,129 @@
+* Configurable protocol scheme for HTTP connections to the upload and ingest API
+
+## Secondary Analysis
+### Cromwell-tools v1.1.1
+* Add on_hold parameter to start_workflow
+* Add workflow validation capability
+* Add Sphinx autodoc and readthedocs hook
+* Bugfix: Allow workflows without labels to be executed
+* Bugfix: Fix docker container build issues
+* Add a new function to query workflows.
+* Add a new function to release workflows from status On Hold to Submitted.
+* A breaking refactoring work on both the CLI and API.
+* More sophisticated unit tests.
+* Support both Python2 and Python3, having tests run on both versions.
+* Updated ReadMe file and docstrings.
+* For /submit endpoint, we support both path to the file and file contents as inputs now.
+* We have a quickstart example Jupyter Notebook now.
+* Update requests to avoid security issues.
+* Fix API calls in CLI.
+* We have accessory tools with cromwell-tools, such as the workflow monitoring script.
+* Other slight improvements and bug fixes.
+* Fix a bug with the service account key that prevents users from using any authentication methods.
+* Fix a bug with /validate command in CLI.
+* Fix a bug with /health endpoint in API.
+* Improve the output of the CLI commands.
+* Update the documentation, quickstart Jupyter notebook, and docstrings.
+* Update the authentication dependency from the deprecated oauth2client to google-auth.
+* Allow no-auth with Cromwell.
+* Improve the help messages of CLI and add shorthands for submit command.
+* Fix an issue that can stop local file from being normally processed.
+* Fix the failing test by cleaning up the Dockerfile that relies on Oracle Java installer and move to use OpenJDK instead.
+* Take the retry policy based on tenacity out from the cromwell-tools for clarity.
+* Add token checker before making the header to avoid token expiration with OAuth.
+* Fix a bug that prevents generating correct pagination parameters for POST /query endpoint.
+* Update the pull-request-template.
+* Fix a bug that prevents submitting a zipped file as a dependency of a workflow.
+
+### Lira v0.19.0
+* Update field name in subscriptions (library_construction_approach -> library_construction_method )
+* Use latest version of cromwell-tools
+
+### Pipeline-tools v0.48.1
+* Update format of the Cromwell url used to fetch metadata
+* Replace deprecated setup.py dependency_links
+* Remove unused Cromwell metadata docker folder
+* Update the adapter workflow for Optimus pipeline, tested with 4k_pbmc dataset.
+* Remove the outputs field from the analysis_process metadata generated by the pipelines.
+* Fix payload for adding file references via the ingestion service API
+
+### Falcon v0.2.0
+* Refactor and cleaning up deployment-related files.
+* Update cromwell-tools version.
+* Update example config.json in README
+
+### Optimus pipeline: v0.4.0
+
+## Data Portal
+### Azul deployed/staging/2019-03-20__08-46
```

Comment:
```suggestion
## Azul 

Version: deployed/staging/2019-03-20__08-46
```

Data Portal and Azul are separate components. They have distinct release engineers. We shouldn't conflate them in the release notes.

Azul version tags are long so it looks better to list them separately.